### PR TITLE
playground: support ticdc for playground

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
support ticdc for playground

### What is changed and how it works?
```
➜  tiup git:(master) ✗ tiup playground --tiflash 0 --ticdc 1
Starting component `playground`: /Users/leoppro/.tiup/components/playground/v1.1.2/tiup-playground --tiflash 0 --ticdc 1
Use the latest stable version: v4.0.5

    Specify version manually:   tiup playground <version>
    The stable version:         tiup playground v4.0.0
    The nightly version:        tiup playground nightly

Playground Bootstrapping...
Start pd instance
Start tikv instance
Start tidb instance
Start ticdc instance
Waiting for tidb 127.0.0.1:4000 ready  ... ⠏ 
Waiting for tidb 127.0.0.1:4000 ready  ... Done
CLUSTER START SUCCESSFULLY, Enjoy it ^-^
To connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
To view the dashboard: http://127.0.0.1:2379/dashboard
To view the Prometheus: http://127.0.0.1:9090
To view the Grafana: http://127.0.0.1:3000


➜  tiup git:(master) ✗ tiup playground scale-out --ticdc 2
Starting component `playground`: /Users/leoppro/.tiup/components/playground/v1.1.2/tiup-playground scale-out --ticdc 2
➜  tiup git:(master) ✗ tiup playground display                                      
Starting component `playground`: /Users/leoppro/.tiup/components/playground/v1.1.2/tiup-playground display
Pid    Role   Uptime
---    ----   ------
40091  pd     31.010579721s
40092  tikv   31.003603491s
40093  tidb   30.998851345s
40094  ticdc  30.992450324s
41704  ticdc  13.818018056s
41705  ticdc  13.813229267s

➜  tiup git:(master) ✗ tiup playground scale-in --pid 41704
Starting component `playground`: /Users/leoppro/.tiup/components/playground/v1.1.2/tiup-playground scale-in --pid 41704
scale in ticdc success
➜  tiup git:(master) ✗ tiup playground display             
Starting component `playground`: /Users/leoppro/.tiup/components/playground/v1.1.2/tiup-playground display
Pid    Role   Uptime
---    ----   ------
40091  pd     15m22.272593345s
40092  tikv   15m22.265617458s
40093  tidb   15m22.260864378s
40094  ticdc  15m22.254463605s
41705  ticdc  15m5.0752401s

```

Release notes:

```release-note
Support TiCDC for playground
```

### CODE REVIEW

File: components/playground/instance/ticdc.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	ticdc.StatusPort = ticdc.Port
+	return ticdc
+}
+
+// NodeID return the node id of pump.
+func (c *TiCDC) NodeID() string {
+	return fmt.Sprintf("ticdc_%d", c.ID)
+}
+
+// Ready return nil when pump is ready to serve.
```
Comment (july2993) [inline components/playground/instance/ticdc.go:60]: pls fix comment pump?

File: components/playground/instance/ticdc.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+			BinPath:    binPath,
+			ID:         id,
+			Dir:        dir,
+			Host:       host,
+			Port:       utils.MustGetFreePort(host, 8300),
+			ConfigPath: configPath,
+		},
+		pds: pds,
+	}
+	ticdc.StatusPort = ticdc.Port
```
Comment (july2993) [inline components/playground/instance/ticdc.go:51]: where is this `StatusPort` used?

File: components/playground/instance/ticdc.go [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+	return ticdc
+}
+
+// NodeID return the node id of pump.
+func (c *TiCDC) NodeID() string {
+	return fmt.Sprintf("ticdc_%d", c.ID)
+}
+
+// Ready return nil when pump is ready to serve.
+func (c *TiCDC) Ready(ctx context.Context) error {
```
Comment (july2993) [inline components/playground/instance/ticdc.go:61]: note this is not used anywhere now

**Review by july2993 — COMMENTED** [review]
rest LGTM

File: components/playground/instance/ticdc.go (hunk 2 above)
Comment (leoppro) [reply components/playground/instance/ticdc.go:48]: this is not used anywhere for now, but I `StatusPort` should be assigned a value

**Comment by leoppro** [comment]
@july2993  @lonng  PTAL again

**Review by july2993 — APPROVED** [review]

**Pull request merged**